ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -58,6 +61,20 @@ export default {
         },
       },
     },
+    {
+      id: 'DelubrumSav Seeker Iron Impact',
+      // This is an early warning on the Verdant Path cast.
+      netRegex: NetRegexes.startsUsing({ source: 'Trinity Seeker', id: '5A99', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Trinität Der Sucher', id: '5A99', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Trinité Soudée', id: '5A99', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'トリニティ・シーカー', id: '5A99', capture: false }),
+      alertText: (data, _, outputs) => outputs.text(),
+      outputStrings: {
+        text: {
+          en: 'Get Behind For Line Stack',
```

Comment:
May want to just say "Line Stack" or "Line AoE" as it is more in line with how cactbot has previously handled similar mechanics. The drawback here is there isn't a marker for it like some of the usage has. I believe Iron Impact targets a random player out of all 48 players.

Reply:
I think "line stack" on its own usually means "look for the line stack, and go to it".  I suspect that most groups stack behind.  I think "stack for line stack" just sounded too awkward, so I went with "get behind".